central/graphql/resolvers/test_setup_utils.go

```diff
@@ -60,7 +60,8 @@ import (
 	"gorm.io/gorm"
```

Comment:
Move this out of graphql and into `pkg/graphql/testutils/`. Importing graphql pkgs into other parts of code will end up causing import cycle in future because graphQL imports almost all central pkgs and will continue import more.

Reply:
hmm...seems like this will require a bit of refactoring. Okay, you can merge this as is. I can TAL when I find time.